Cargo.toml

```diff
@@ -79,7 +79,10 @@ itertools = { default-features = false, version = "0.14.0" }
 convert_case = { default-features = false, version = "0.8.0" }
 serde_json = { default-features = false, version = "1.0.143" }
 trybuild = { default-features = false, version = "1.0.110" }
-typed-builder = { default-features = false, version = "0.21.2" }
+# typed-builder = { default-features = false, version = "0.21.2" }
+# typed-builder-macro = { default-features = false, version = "0.21.0" }
+typed-builder = { git = "https://github.com/zakstucke/rust-typed-builder.git", rev = "3bc55429a0036d0143ed96f9e48715def754f99d" }
```

Comment:
Note: we should remember to revert these two when the upstream PR is merged. (We can't publish to crates.io with git dependencies)